src/main.rs

```diff
@@ -28,13 +29,26 @@ fn main() {
         .help("The path that the prompt should render for")
         .takes_value(true);
 
-    let matches = App::new("Starship")
+    let shell_arg = Arg::with_name("shell")
+        .value_name("SHELL")
+        .help(
+            "The name of the currently running shell\nCurrently supported options: bash, zsh, fish",
```

Comment:
```suggestion
            "The name of the current running shell\nCurrently supported options: bash, zsh, fish",
```

Reply:
Also, do you want to display the currently running shell? There is text there ... but nothing else

On second thought though ... Grammarly is telling me that "current running shell" is incorrect and it should be "currently running shell" ... so whichever one sounds better to you!